pkg/apiserver/apiserver_test.go

```diff
@@ -961,7 +1068,7 @@ func TestCreateTimeout(t *testing.T) {
 
 	simple := &Simple{Other: "foo"}
 	data, _ := codec.Encode(simple)
-	itemOut := expectApiStatus(t, "POST", server.URL+"/prefix/version/foo?timeout=4ms", data, http.StatusAccepted)
+	itemOut := expectApiStatus(t, "POST", server.URL+"/prefix/version/foo?timeout=4ms", data, 504)
```

Comment:
Could you please define a constant for 504 here:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/errors/errors.go#L30